### DISCUSSION TITLE
Outdoor temperature display precision

### DISCUSSION THREAD

**Discussion by Achronite** [Blackymas/NSPanel_HA_Blueprint#317] [Polls]
The outdoor temperature on the main page is currently displayed as a rounded whole number.  This value is inconsistent with that shown for the indoor temperature, which displays with 1 decimal point precision.

This poll is to ask the community on their views on changing this precision value, so that when the 'Outdoor Temperature Sensor' is a value that contains decimal places it should be shown on the display.

See also #316 for original issue

**Comment by Blackymas** [comment]
Is changed in v3.2 
Please update and test :)

https://github.com/Blackymas/NSPanel_HA_Blueprint/releases/tag/v.3.2
